tests/chart_tests/test_global_ingress_annotation.py

```diff
@@ -0,0 +1,32 @@
+import pytest
+
+from tests import supported_k8s_versions
+from tests.chart_tests.helm_template_generator import render_chart
+
+
+@pytest.mark.parametrize(
+    "kube_version",
+    supported_k8s_versions,
+)
+class TestGlobabIngressAnnotation:
+    def test_global_ingress_with_astronomer_ingress(self, kube_version):
+        """Test global ingress annotation for platform ingress ."""
+        docs = render_chart(
+            kube_version=kube_version,
+            values={"global": {"extraAnnotations": {"route.openshift.io/termination": "passthrough"}}},
+            show_only=[
+                "charts/alertmanager/templates/ingress.yaml",
+                "charts/grafana/templates/ingress.yaml",
+                "charts/kibana/templates/ingress.yaml",
+                "charts/prometheus/templates/ingress.yaml",
+                "charts/astronomer/templates/ingress.yaml",
+                "charts/astronomer/templates/registry/registry-ingress.yaml",
+            ],
```

Comment:
Can we switch this to a dynamic find of all ingress files? something like `sorted([str(x.relative_to(GIT_ROOT)) for x in Path(GIT_ROOT).rglob('*ingress*.yaml')])` so we ensure we always get all ingress files, even if we add a new ingress file?

Reply:
```suggestion
            show_only=sorted([str(x.relative_to(GIT_ROOT)) for x in Path(GIT_ROOT).rglob('*ingress*.yaml')]),
```